buildchain/buildchain/docker_command.py

```diff
@@ -0,0 +1,214 @@
+# coding: utf-8
+
+"""Provides Python commands for Docker Python client.
+
+Supply callable objects with prepared configuration for use
+with `doit`'s Python actions.
+"""
+
+import copy
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import docker                            # type: ignore
+from docker.types import Mount           # type: ignore
+import doit                              # type: ignore
+from doit.exceptions import TaskError    # type: ignore
+
+from buildchain import constants
+
+
+DOCKER_CLIENT : docker.DockerClient = docker.from_env()
+
+
+# The call method is not counter as a public method
+# pylint: disable=too-few-public-methods
+class BuildCommand:
+    """Docker build callable"""
```

Comment:
This doesn't tell much, could say something like "A class exposing the `docker build` command".

Dittto for the other classes.